src/test/java/uk/gov/digital/ho/pttg/api/RequestHeaderDataTest.java

```diff
@@ -244,6 +243,30 @@ public void proceed_whenRequestDurationBeyondThreshold_log() {
                         HMRC_INSUFFICIENT_TIME_TO_COMPLETE)));
     }
 
+    @Test
+    public void retryCount_notPassed_returnsNegative() {
+        given(mockHttpServletRequest.getHeader(RETRY_COUNT_HEADER)).willReturn("");
```

Comment:
getHeader returns null when the header is not present so we need a test for that